tendermint-lite/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "tendermint-lite"
+version = "0.1.0"
+authors = ["Ethan Buchman <ethan@coinculture.info>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tendermint = { path = "../tendermint" }
+tokio = "0.2"
```

Comment:
Cool that there is (again) a light client crate now.  Is the plan to move everything from `tendermint/src/lite` into this crate too?

Reply:
I'm starting to think the `tendermint/src/lite` should be its own crate and the non main stuff here should be in the `tendermint` crate

Reply:
OK, I see. For binaries it is good practice to add the `Cargo.lock` to the repo, too.